src/Plugin.php

```diff
@@ -42,6 +42,10 @@ public static function init() {
     if (is_admin()) {
       add_action('admin_enqueue_scripts', __CLASS__ . '::admin_enqueue_scripts');
```

Comment:
1. This should be performed in an Admin class instead.

1. All Plugin/frontend code seems to depend on plugins woocommerce and ACF to be installed and active. I would suggest to add an early return to Plugin::init() in case the functions register_field_group() and is_product_category() do not exist.